client/src/main/java/org/asynchttpclient/netty/NettyResponseFuture.java

```diff
@@ -47,24 +45,18 @@
  * 
  * @param <V> the result type
  */
-public final class NettyResponseFuture<V> extends AbstractListenableFuture<V> {
+public final class NettyResponseFuture<V> implements ListenableFuture<V> {
```

Comment:
So `AbstractListenableFuture` and `RunnableExecutorPair` can be removed, as they are replaced with `CompletableFuture#whenCompleteAsync`, right?